### PR TITLE
Add confirmation delete project and client

### PR DESCRIPTION
**Pull request by fitmariusz**
Create confirmation after Delete Project and Client.

### CODE REVIEW

File: src/commands/client.js [hunk 1]
```diff
@@ -49,9 +49,18 @@ const deleteClient = async () => {
...
 
-  const resp = await clientService.delete(clients.find(c => c.name === name));
-
-  return resp;
+  const decision = await inquirer.input({
+    default: 'no',
+    message: `Are you sure to delete client: ${name} ? (yes/no)`,
+  });
+  
+  if(decision.toLowerCase()==='yes' | decision.toLowerCase()==='y')
```
Comment (KamilMr) [inline src/commands/client.js:57]: You should use `||` instead of `|`.

Comment (KamilMr) [inline src/commands/client.js:57]: Use spaces between elements. For example: 
```
if (decision === 'yes' || decision === 'y') 
```

Comment (KamilMr) [reply src/commands/client.js:57]: When you use `if` statement have `{` on the same line as if - if possible.

File: src/commands/project.js [hunk 2]
```diff
@@ -113,9 +113,19 @@ const deleteProject = async () => {
...
+  const decision = await inquirer.input({
+    default: 'no',
+    message: `Are you sure to delete project: ${name} ? (yes/no)`,
+    
+  });
+  
+  if(decision.toLowerCase()==='yes' | decision.toLowerCase()==='y')
+  {
+    const project = data.find(c => c.name === name);
+    console.log('Project delete..')
```
Comment (KamilMr) [inline src/commands/project.js:125]: Don't need that console.log

File: src/commands/project.js [hunk 3]
```diff
@@ -113,9 +113,19 @@ const deleteProject = async () => {
...
-  const project = data.find(c => c.name === name);
-
-  await projectService.delete(project);
+  const decision = await inquirer.input({
+    default: 'no',
+    message: `Are you sure to delete project: ${name} ? (yes/no)`,
+    
+  });
+  
+  if(decision.toLowerCase()==='yes' | decision.toLowerCase()==='y')
```
Comment (KamilMr) [inline src/commands/project.js:122]: Previous comments applies to this place as well

File: src/commands/client.js [hunk 4]
```diff
@@ -49,9 +49,18 @@ const deleteClient = async () => {
 
   if (name === 'Stop') return;
 
-  const resp = await clientService.delete(clients.find(c => c.name === name));
-
-  return resp;
+  const decision = await inquirer.input({
```
Comment (KamilMr) [inline src/commands/client.js:52]: `confirmation` makes more sense?

File: src/commands/client.js [hunk 5]
```diff
@@ -49,9 +49,18 @@ const deleteClient = async () => {
...
+    message: `Are you sure to delete client: ${name} ? (yes/no)`,
+  });
+  
+  if(decision.toLowerCase()==='yes' | decision.toLowerCase()==='y')
+  {
+    const resp = await clientService.delete(clients.find(c => c.name === name));
+    return resp;
+  }
+  
+  return true;
```
Comment (KamilMr) [inline src/commands/client.js:63]: I think returning true does not change anything, does it?

**Review by KamilMr — CHANGES_REQUESTED** [review]
Hej, 
Almost there. 
Few things to change inside pull request. :)

File: src/commands/client.js (hunk 1 above)
Comment (KamilMr) [reply src/commands/client.js:57]: - [ ] Also here force user user to write only:  yes, y, no, n 

At the moment when he types something else, it is also valid.

**Comment by fitmariusz** [comment]
Hi I Addition of validation during confirmation of PROJECT DELETE and CLIENT DELETE

File: src/commands/client.js [hunk 6]
```diff
@@ -49,9 +48,27 @@ const deleteClient = async () => {
...
+      let resp;
+      if ((resp = isConfirmationInvalid(input.toLowerCase()))) return resp;
+
+      return true;
 
-  return resp;
+    },
+  });
+  
+  if(confirmation.toLowerCase()==='yes' || confirmation.toLowerCase()==='y')
```
Comment (KamilMr) [inline src/commands/client.js:67]:
```suggestion
  if (confirmation.toLowerCase() === 'yes' || confirmation.toLowerCase () === 'y' )
```

**Pull request merged**